video2dataset/worker.py

```diff
@@ -199,6 +203,15 @@ def data_generator():
                             meta["key"],
                             sample_data[caption_indice] if caption_indice is not None else None,
                             meta,
+                            format_type="video",
+                        )
+                    if aud_stream is not None:
```

Comment:
instead of doing it like this can't we just pass streams into here the sample writer? The sample writer should know all the streams to expect because you pass in encode_formats to it so can't we just write those all together?